test/api/link/find-subgroup-by-groupId.test.js

```diff
@@ -0,0 +1,48 @@
+const test = require('tape')
+const { Server } = require('../../helpers')
+const { isCloakedMsg: isGroup } = require('ssb-ref')
+
+test('tribes.subtribe.findByGroupId', t => {
```

Comment:
if I were you I would start getting into writing async/ await tests with promisify.
IMO it makes these sorts of tests a lot easier to write and read.
would not change this, but think about it for next time

Reply:
I generally just follow the pattern of whats already in existing tests. In this case, i wrote the same pattern that was in another similar test. Will think about this feedback next time though